Lesson_204V2/FullSolution/MainPage.xaml.cs

```diff
@@ -0,0 +1,196 @@
+﻿using System;
+using System.Diagnostics;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using Windows.Devices.Adc;
+using Windows.Media.SpeechSynthesis;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Navigation;
+
+using Microsoft.IoT.AdcMcp3008;
+// The Blank Page item template is documented at http://go.microsoft.com/fwlink/?LinkId=402352&clcid=0x409
+
+namespace Lesson_204V2
+
+{
+    /// <summary>
+    /// An empty page that can be used on its own or navigated to within a Frame.
+    /// </summary>
+    public sealed partial class MainPage : Page
+    {
+        // Use for configuration of the MCP3008 class voltage formula
+        const float ReferenceVoltage = 5.0F;
+
+        // Values for which channels we will be using from the ADC chip
+        const byte LowPotentiometerADCChannel = 0;
+        const byte HighPotentiometerADCChannel = 1;
+        const byte CDSADCChannel = 2;
+
+        // Some strings to let us know the current state.
+        const string JustRightLightString = "Ah, just right";
+        const string LowLightString = "I need a light";
+        const string HighLightString = "I need to wear shades";
+
+        // Some internal state information
+        enum eState { unknown, JustRight, TooBright, TooDark};
+        eState CurrentState = eState.unknown;
```

Comment:
Are these public or private?  Below you clearly declare synthesizer as private.  Why not adcController and the AdcChannels?